ext_autoload.php

```diff
@@ -1,13 +1,232 @@
 <?php
-// autoloading seems to be pretty useless for static classes... :-(
-$versionParts = explode('.', TYPO3_version);
-$extensionPath = (intval($versionParts[0]) >= 6) ?
-	\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath('rn_base') :
-	t3lib_extMgm::extPath('rn_base');
+// DO NOT CHANGE THIS FILE! It is automatically generated by extdeveval::buildAutoloadRegistry.
+// This file was generated on 2016-07-01 20:00
 
+$extensionPath = t3lib_extMgm::extPath('rn_base');
```

Comment:
Warum ist die Versionsprüfung raus? In 6.2 wird das doch zu Meldungen im deprecation-Log führen, oder?